### PR TITLE
Data: project addition by Deepanshu #981

### PR DESCRIPTION
**Pull request by Deepanshu0703**
## Related Issue

This is issue number #981 

## Description

Adde new project of Google keeper clone using ReactJs

### CODE REVIEW

File: src/DB/projects.json [hunk 1]
```diff
@@ -2005,5 +2005,23 @@
...
+      "YouTube": "",
+      "Twitter": ""
+    },
+    "Projects": [
+      {
+        "link": "https://github.com/Deepanshu0703/google-keeper-clone",
+        "title": "Google Keeper Clone",
+        "description": "This is a clone of google keeper app. It is a note taking app where you can add, delete and edit notes.",
+        "tech": ["ReactJs", "CSS"]
+      }
```
Comment (priyankarpal) [inline src/DB/projects.json:2024]: hey just add the code to the previous code. you have already added your projects here. you don't need to add whole code multiple times

**Review by priyankarpal — CHANGES_REQUESTED** [review]

File: src/DB/projects.json (hunk 1 above)
Comment (Deepanshu0703) [reply src/DB/projects.json]: Oh Sorry for that. Now corrected it

**Comment by Deepanshu0703** [comment]
Changed the title

**Review by priyankarpal — APPROVED** [review]
Thank you for contributing your projects here 👏🏻 🎉

**Pull request merged**